### PR TITLE
SC-1534

### PR DESCRIPTION
**Pull request by anmol2302**
<b>Ticket Ref:</b> [SC-1534](https://project-sunbird.atlassian.net/browse/SC-1534)
**Release:** 2.6.0
New Code Coverage: ` Coverage           (Old) 26.06%   (New)31.48%   +5.42%  🆙  ` (MW)
Updates: Added Test Cases for Following Files

1. AddressManagamentActor
2. JobProfileActor
3. Utility Classes
4. UserOrgManagementActor

### CODE REVIEW

File: actors/common/src/main/java/org/sunbird/common/quartz/scheduler/SchedulerManager.java [hunk 1]
```diff
@@ -273,7 +273,8 @@ private void scheduleShadowUser(String identifier) {
     ProjectLogger.log(
         "SchedulerManager:scheduleShadowUser: schedule time is : " + shadowUserTime,
         LoggerEnum.INFO.name());
-     Trigger migrateShadowUserTrigger =
+    shadowUserTime="0 0/5 * 1/1 * ? *";
```
Comment (manzarul) [inline actors/common/src/main/java/org/sunbird/common/quartz/scheduler/SchedulerManager.java]: remove shadowUserTime="0 0/5 * 1/1 * ? *";

File: actors/common/src/main/java/org/sunbird/learner/actors/user/service/UserService.java [hunk 2]
```diff
@@ -64,4 +57,12 @@ public void checkKeyUniqueness(String key, String value, boolean isEncrypted) {
     }
   }
 
+  public CassandraOperation getCassandraOperation(){
```
Comment (manzarul) [inline actors/common/src/main/java/org/sunbird/learner/actors/user/service/UserService.java]: getCassandraOperation method is called by only this class , so scope can be private  
same for getEncryptionService

File: actors/common/src/main/java/org/sunbird/common/quartz/scheduler/SchedulerManager.java (hunk 1 above)
Comment (anmol2302) [reply actors/common/src/main/java/org/sunbird/common/quartz/scheduler/SchedulerManager.java]: done

File: actors/common/src/main/java/org/sunbird/learner/actors/user/service/UserService.java (hunk 2 above)
Comment (anmol2302) [reply actors/common/src/main/java/org/sunbird/learner/actors/user/service/UserService.java]: done

**Pull request merged**